packages/tools/src/account/unlock.ts

```diff
@@ -0,0 +1,360 @@
+import {
+  Cell,
+  CellDep,
+  Hash,
+  HexString,
+  utils,
+  WitnessArgs,
+  core as baseCore,
+  Transaction,
+  OutPoint,
+  Script,
+} from "@ckb-lumos/base";
+import {
+  parseAddress,
+  sealTransaction,
+  TransactionSkeleton,
+} from "@ckb-lumos/helpers";
+import { CellCollector, Indexer } from "@ckb-lumos/indexer";
+import { deploymentConfig } from "../modules/deployment-config";
+import {
+  ROLLUP_TYPE_HASH,
+  ROLLUP_TYPE_SCRIPT,
+} from "../modules/godwoken-config";
+import { asyncSleep, privateKeyToCkbAddress } from "../modules/utils";
+import { exit } from "process";
+import { core, normalizer } from "@godwoken-examples/godwoken";
+import { normalizers, Reader } from "ckb-js-toolkit";
+import { common } from "@ckb-lumos/common-scripts";
+import { key } from "@ckb-lumos/hd";
+import { Command } from "commander";
+import { initConfigAndSync } from "./common";
+import { getConfig } from "@ckb-lumos/config-manager";
+import { RPC } from "@ckb-lumos/rpc";
+
+async function unlock(
+  privateKey: HexString,
+  indexer: Indexer,
+  rpc: RPC,
+  sudtScript?: Script,
+  retryTime = 10
+) {
+  try {
+    for (let i = 0; i < retryTime; i++) {
+      if (i > 0) {
+        console.log("-".repeat(15) + ` Retry ${i} ` + "-".repeat(15));
+      }
+
+      const isRetry = await unlockInner(privateKey, indexer, rpc, sudtScript);
```

Comment:
maybe `_unlock` is a better name

Reply:
Prefix with `_` usually means it's an unused variable.